GTG/gtk/editor/editor.py

```diff
@@ -252,6 +344,10 @@ def init_dimensions(self):
     # Refreshtext is whether or not we should refresh the TaskView
     # (doing it all the time is dangerous if the task is empty)
     def refresh_editor(self, title=None, refreshtext=False):
+        '''
+        TODO(jakubbrindza): Repair refresh functionality for tags and parents
+        (a.k.a. adjust the commented out code for current needs)
+        '''
```

Comment:
This is not true, and you already solved this by deleting them! Kill it with :fire:!